src/services/user/services/AdminUsers.js

```diff
@@ -74,6 +75,9 @@ class AdminUsers {
 			const [currentUser, roles] = await Promise.all([getCurrentUser(currentUserId), getRoles()]);
 			const { schoolId } = currentUser;
 
+			const currentSchool = await this.app.service('schools').get(schoolId);
```

Comment:
You can populate the school  in getCurrentUser. The helper only fetch the roles and school data of current user. It should rename. Destructure the result will also fine. [{roles, schoolId}, roles]

Reply:
via the populate I dont get the year info im looking for, since its added by a hook.